### PR TITLE
Dialogs review

### PR DESCRIPTION
**Pull request by lnewson**
Reviewed the dialogs page and made any relevant updates/fixes to incorrect or out of date information. Please feel free to make any other additional changes you want :)

### CODE REVIEW

File: ui-components/dialog.md [hunk 1]
```diff
@@ -15,48 +15,49 @@ A dialog is a TinyMCE UI component. Dialogs have their own dialog components whi
 
 * **Display complex information** - These dialogs can display complex information by using layouts components like tabs or columns to help present information to the user (e.g., help dialog or special characters dialog are tabbed dialogs).
 
-* **Interactive dialogs** - These dialogs use web forms to collect interaction data and then apply the data (e.g.: search and replace dialog, uses an input field. Where the input text will be used as the search key. Another example is, special characters or character map dialogs use typeaheads to dynamically narrow down matches as you type)
+* **Interactive dialogs** - These dialogs use web forms to collect interaction data and then apply the data (e.g.: search and replace dialog, uses an input field, where the input text will be used as the search key. Another example is, special characters or character map dialogs use typeaheads to dynamically narrow down matches as you type)
```
Comment (blackcathikari) [inline ui-components/dialog.md]: We should probably make this "apply the data. For example, search and replace dialog...". Having so much in brackets seems odd.

File: ui-components/dialog.md [hunk 2]
```diff
@@ -15,48 +15,49 @@ A dialog is a TinyMCE UI component. Dialogs have their own dialog components whi
...
+   buttons: [] // A list of button configurations the dialog will have.
 }
 ```
 
 ## Dialog configuration framework
 
 A Dialog configuration framework has three main parts:
 
-* **Title** This is the title of a dialog.
+* **Title** This is the ti
```
Comment (blackcathikari) [inline ui-components/dialog.md]: A newline slipped in somehow ;)

File: ui-components/dialog.md [hunk 3]
```diff
@@ -70,34 +71,40 @@ var tabPanelConfig = {
   ]
 };
 ```
-**Tabs:** These are an array of tab configurations. Each tab has a title which is used to reference the tab. The items property in the tab configuration takes a list of components and works the same way as a Panel. A tab can be programmatically be switched by calling `dialogApi.showTab(‘title’)`. For example, the dialog that appears as a result of Help plugin is usually formatted in tab panels.
+**Tabs:** These are an array of tab configurations. Each tab has a title which is used to reference the tab. The items property in the tab configuration takes a list of components and works the same way as a Panel. A tab can be programmatically be switched by calling `dialogApi.showTab('title')`. For example, the dialog that appears as a result of the Help plugin is usually formatted in tab panels.
```
Comment (blackcathikari) [inline ui-components/dialog.md]: Backticks around items in the third sentence.

File: ui-components/dialog.md [hunk 4]
```diff
@@ -125,23 +132,23 @@ const dialogConfig = {
...
-     items: [] //A list of UI component configurations the dialog will have.
+     items: [] // A list of UI component configurations the dialog will have.
    },
-   buttons: [] //A list of button configurations the dialog will have.
+   buttons: [] // A list of button configurations the dialog will have.
 }
 ```
 
-Using the above example, calling `tinymce.activeEditor.windowManager.open(dialogConfig)` will create a dialog with the title just a title, an empty body, and an empty footer without buttons.
+Using the above example, calling `tinymce.activeEditor.windowManager.open(dialogConfig)` will create a dialog with the title **just a title**, an empty body, and an empty footer without buttons.
```
Comment (blackcathikari) [inline ui-components/dialog.md]: Capitals in "just a title" to match the example?

File: ui-components/dialog.md (hunk 2 above)
Comment (lnewson) [reply ui-components/dialog.md]: Doh... thanks for the catch!

**Comment by lnewson** [comment]
Thanks for the review @blackcathikari! I think I've fixed everything up :smile:

**Pull request merged**